src/modules/UI/scenes/EditToken/EditToken.ui.js

```diff
@@ -184,8 +184,12 @@ export default class EditToken extends Component<EditTokenComponentProps, State>
   }
 
   onChangeCurrencyCode = (input: string) => {
+    const forcedUpperCase = input.toUpperCase()
+    /* forcedUpperCase needed to defend against React Native bug
+      https://github.com/facebook/react-native/issues/11776
+    */
     this.setState({
-      currencyCode: input.substring(0, 5)
+      currencyCode: forcedUpperCase.substring(0, 5)
```

Comment:
Please address the problem as close to the root as possible, so if it ever gets fixed, we only have to remove a single shim.
`src/components/materialWrappers/Input.js`
Comment out the default prop for `keyboardType`, and include a comment/link to the github pull request that identifies the bug